src/main/java/tecna/Main.java

```diff
@@ -0,0 +1,37 @@
+package tecna;
+
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Tecna using FXML.
+ */
+public class Main extends Application {
+    private Tecna tecna = new Tecna("data/tecna.json");
+
+    @Override
+    public void start(Stage stage) {
```

Comment:
could make the name more precise by naming it startApplication or something like that but start is still quite clear to me so if you feel like putting extra effort then can be more precise i guess